src/main/java/ch/geowerkstatt/ilivalidator/extensions/functions/UnionIoxPlugin.java

```diff
@@ -65,7 +65,7 @@ private IomObject union(UnionSurfaceKey key) {
                 .toArray(MultiPolygon[]::new);
 
         Geometry geometryCollection = new GeometryFactory().createGeometryCollection(polygons);
-        Geometry unionGeometry = geometryCollection.union();
+        Geometry unionGeometry = geometryCollection.buffer(0);
```

Comment:
REQ: Kommentar weshalb nicht .union verwendet wird. Ev. gleich ein Link auf ein Issue in welcher Version von JTS dies korrigiert wird.